atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -1,22 +1,20 @@
 import uuid
```

Ruff (0.12.2)

Invalid module name: 'PipelineWorkflows'

(N999)